online/src/main/scala/ai/chronon/online/MetadataStore.scala

```diff
@@ -139,6 +139,44 @@ class MetadataStore(kvStore: KVStore, val dataset: String = ChrononMetadataKey,
     },
     { join => Metrics.Context(environment = "join.meta.fetch", join = join) })
 
+  // Validate whether the join exists in the active groupBy list saved in kv store
+  def validateJoinExist(team: String, name: String): Boolean = {
+    val activeJoinList: Try[Seq[String]] = getJoinListByTeam(team)
```

Comment:
I might miss some context here. How do we know the join list is all active?

Reply:
Hi @pengyu-hou We are uploading the active entity list by each team in metadata-upload job. The k-v pair is like:
`joins/team_a -> [joins/team_a/join_a.v1, joins/team_a/join_a.v2]` When users are fetching we fetch the active entity list by team first from the same k-v store where we save the groupByServingInfo and joinConf, check whether the entity is in the list and then do the fetching.

Reply:
For Airbnb specifically the process is done in the two tasks here: https://airflow-1-10.d.musta.ch/graph?dag_id=chronon_metadata_upload

Reply:
I see. So the value is always the active list. Is my understanding correct?

Reply:
@pengyu-hou  Yep! It is an active list which get refreshed every 15 minutes. Since it is a list, the removed item will not be in the list once removed.